medusa/notifiers/emailnotify.py

```diff
@@ -287,39 +279,49 @@ def _generate_recipients(show):  # pylint: disable=too-many-branches
                                     addrs.append(addr)
 
         addrs = set(addrs)
-        logger.log('Notification recipients: {}'.format(addrs), logger.DEBUG)
+        log.debug('Notification recipients: {0}', addrs)
         return addrs
 
     def _sendmail(self, host, port, smtp_from, use_tls, user, pwd, to, msg, smtpDebug=False):  # pylint: disable=too-many-arguments
-        logger.log('HOST: {}; PORT: {}; FROM: {}, TLS: {}, USER: {}, PWD: {}, TO: {}'.format(
-            host, port, smtp_from, use_tls, user, pwd, to), logger.DEBUG)
+        log.debug(
+            'HOST: {host}; PORT: {port}; FROM: {sender}, TLS: {tls},'
+            ' USER: {user}, PWD: {password}, TO: {recipient}', {
+                'host': host,
+                'port': port,
+                'sender': smtp_from,
+                'tls': use_tls,
+                'user': user,
+                'password': pwd,
+                'recipient': to,
+            }
+        )
         try:
             srv = smtplib.SMTP(host, int(port))
-        except Exception as e:
-            logger.log('Exception generated while sending e-mail: ' + str(e), logger.WARNING)
+        except Exception as error:
+            log.warning('Exception generated while sending e-mail: {0}', error)
             # logger.log(traceback.format_exc(), logger.DEBUG)
-            self.last_err = '{}'.format(e)
+            self.last_err = '{}'.format(error)
```

Comment:
Yeah missed adding that one, good catch... after doing so many you tend to miss some :)